.github/workflows/terraform-plan-apply.yml

```diff
@@ -269,6 +271,23 @@ jobs:
           echo "tenant_id=$(yq eval '.tenant_id' ${{ matrix.definition_path }})" >> $GITHUB_ENV
           echo "ui_client_id=$(yq eval '.ui_client_id' ${{ matrix.definition_path }})" >> $GITHUB_ENV
 
+      # Prepare versions json
+      - name: Prepare versions json
```

Comment:
what do you think about writing this file into the assets directory which is anyway copied as a whole?

Reply:
At this point in the github action flow, the UI zip has not yet been wget-ed.   I could move the action of preparing the json to later in the flow, but I think it's better to "fail fast" so anything involving parsing config files happens at the beginning?

Reply:
We spoke on another thread you explained you meant into matrix.trial_dir.   Good idea,  I've made this change and tested it works in this deployment: https://sakate760nuidev.z20.web.core.windows.net/